packages/core/src/shouldIntercept.ts

```diff
@@ -7,6 +7,10 @@ export default function shouldIntercept(event: KeyboardEvent): boolean {
     (isLink && event.altKey) ||
     (isLink && event.ctrlKey) ||
     (isLink && event.metaKey) ||
-    (isLink && event.shiftKey)
+    (isLink && event.shiftKey) ||
+    // The auxiliary button (middle button) was pressed. All major browsers
+    // except Safari trigger the `auxclick` instead of `click`, so this is to
+    // ensure we do not intercept middle clicks in Safari.
+    (isLink && 'button' in event && event.button === 1)
```

Comment:
Should this be `event.button !== 0` instead? (as in "is not left click")

Reply:
That makes sense! I was erring on the side of caution here by surgically targeting the known edge case, but realistically there's probably not much risk expanding to `!== 0`.